.github/workflows/automerge_scan.yml

```diff
@@ -0,0 +1,58 @@
+name: AutoMergeScan
+on:
+  schedule:
+    - cron: '31 */2 * * *'
+  workflow_dispatch:
+
+jobs:
+  automerge_scan:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Debug
+      env:
+        TOKEN: ${{ secrets.TOKEN }}
+      run: |
+        set -e
+
+        echo ${TOKEN} | gh auth login --with-token
+        gh pr list -R sonic-net/sonic-buildimage -A mssonicbld --json additions,assignees,author,baseRefName,body,changedFiles,closed,closedAt,comments,commits,createdAt,deletions,files,headRefName,headRepository,headRepositoryOwner,id,isCrossRepository,isDraft,labels,latestReviews,maintainerCanModify,mergeCommit,mergeStateStatus,mergeable,mergedAt,mergedBy,milestone,number,potentialMergeCommit,projectCards,reactionGroups,reviewDecision,reviewRequests,reviews,state,statusCheckRollup,title,updatedAt,url > prs.log
```

Comment:
We should only take action for the PRs not merged in long time, for example, the PR createdAt >= <specific value>